packages/dbgpt-core/src/dbgpt/rag/text_splitter/tests/test_splitters.py

```
@@ -63,3 +65,26 @@ def test_character_text_splitter_empty_doc() -> None:
+    Guards the spacy>=3.8 upgrade (Python 3.13 support): verifies the public
+    spacy API used by SpacyTextSplitter (spacy.load / nlp(text).sents) still
+    works. Skips when spacy or the en pipeline is unavailable so the suite
+    stays green without the optional ``rag`` extra installed.
+    """
+    pytest.importorskip("spacy", reason="spacy (rag extra) not installed")
+
+    from dbgpt.rag.text_splitter.text_splitter import SpacyTextSplitter
+
+    try:
+        splitter = SpacyTextSplitter(pipeline="en_core_web_sm", chunk_size=1000)
+    except Exception as e:  # pragma: no cover - depends on model download
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)